app/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'com.android.application'
 
 android {
-    def versionCode = 0
+    def newVersionCode = 0
 	if (System.env.BRANCH_NAME) {
```

Comment:
I feel we can remove this check as file is now having same name always.

Reply:
If we would remove this condition, then local builds (outside Jenkins) would update the version number, which would be wrong.

Reply:
Ok. got it.